src/biokbase/narrative/jobs/job.py

```diff
@@ -274,11 +296,11 @@ def was_terminal(self):
             return self._acc_state.get("status") in TERMINAL_STATUSES
 
     def is_terminal(self):
-        self.state()
+        self.refresh_state()
```

Comment:
I don't think this function (`is_terminal`) is called anywhere apart from in tests. There doesn't seem to be much point in keeping it around.